catalog-info.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: avalanche-cli
+  description: |
+    Avalanche-CLI is a command-line interface for building, deploying, and maintaining Avalanche
+    Subnets. It can be used to prototype and test Subnets, as well as to deploy them to the
+    Avalanche network. Avalanche-CLI is available for Linux and Mac, and is open source and
+    available on GitHub.
+  annotations:
+    backstage.io/techdocs-ref: dir:.
+    github.com/project-slug: ava-labs/avalanche-cli
+spec:
+  type: cli
+  owner: platform-tooling
+  lifecycle: production
+  dependsOn:
+    - component:avalanchego
```

Comment:
is this relevant? were are the other components specs to be found by the tools?

Reply:
yeah, I want to start tracking all these dependencies